README.md

```diff
@@ -1,18 +1,17 @@
-**Starting point**: Spreadsheet with logic for helping survey teams, like Heatsmart Alliance coaches via the [sibling project](https://github.com/codeforboston/urban-league-heat-pump-accelerator), to select an appropriately sized heat pump unit (by BTU/hr) for a given household. Current spreadsheet inputs include things like historic energy usage, household thermostat setting, and yearly weather in their zipcode.
+**Goal**: Help heat pump survey teams pick the right heat pump size for a given house. This app would help calculate how many BTUs/hr a house needs in its particular geographical location. Eventually, the tool should store the data for each home to analyze general trends.
 
-This app is based on Excel spreadsheet formulas. See [this Slack message](https://cfb-public.slack.com/archives/C0563F96JSG/p1684285954706029) to download a Zip file of that original Excel, or make a request for a copy from HeatSmart at [this link](https://heatsmartalliance.org/coaching-tools/heat-load-analysis/):
+**Users**: Teams like the Heatsmart Alliance coaches via the [sibling project](https://github.com/codeforboston/urban-league-heat-pump-accelerator).
 
-**Desired**: "Help the helpers" app which assists the heat pump survey teams to fill an app version of the spreadsheet. Eventually, the tool should persist a home profile for aggregation or later reference.
+**Current method**: Excel spreadsheets with formulas that make the calculations. The current spreadsheet inputs include things like historic energy usage, current house thermostat settings, and yearly weather in the house's zipcode. See [this Slack message](https://cfb-public.slack.com/archives/C0563F96JSG/p1684285954706029) to download a Zip file of that original Excel, or make a request for a copy from HeatSmart at [this link](https://heatsmartalliance.org/coaching-tools/heat-load-analysis/):
 
 ## Ideas for Teams
 
-
 ### UI/UX and/or Design
 This team will work with the other teams to match the interface and necessary research together.
 
 ### Frontend
 
-This team will plan and build the user interface. Possible solutions span Vue and React, Figma or your other favorite tools. 
+This team will plan and build the user interface. Possible solutions span Vue and React, Figma or your other favorite tools. It will also get information from the weather station API that the rules engine can use for its calculations. In future, it may work on permissions and persistence of household data.
```

Comment:
`nitpick`: add a "the"

```suggestion
This team will plan and build the user interface. Possible solutions span Vue and React, Figma or your other favorite tools. It will also get information from the weather station API that the rules engine can use for its calculations. In the future, it may work on permissions and persistence of household data.
```